en/SUMMARY.md

```diff
@@ -158,18 +158,19 @@
         * [MindRacer 210](flight_controller/mindracer210.md)
         * [NanoMind 110](flight_controller/nanomind110.md)
   * [Omnibus F4 SD](flight_controller/omnibus_f4_sd.md)
-  * [Snapdragon Flight](flight_controller/snapdragon_flight.md)
-    * [Hardware Setup Example](flight_controller/snapdragon_flight_hardware_example_setup.md)
-    * [Developer Environment Installation](flight_controller/snapdragon_flight_dev_environment_installation.md)
-    * [Snapdragon Software Installation](flight_controller/snapdragon_flight_software_installation.md)
-    * [Configuration](flight_controller/snapdragon_flight_configuration.md)
-    * [Snapdragon Advanced](flight_controller/snapdragon_flight_advanced.md)
-  * [Intel® Aero Ready to Fly Drone](flight_controller/intel_aero.md)
   * [Raspberry Pi 2/3 Navio2](flight_controller/raspberry_pi_navio2.md)
   * [OcPoC-Zynq Mini](flight_controller/ocpoc_zynq.md)
-  * [Crazyflie 2.0](flight_controller/crazyflie2.md)
-  * [Parrot Bebop](flight_controller/bebop.md)
   * [BeagleBone Blue](flight_controller/beaglebone_blue.md)
+  * [Complete Vehicles](flight_controller/complete_vehicles.md)
+    * [Snapdragon Flight](flight_controller/snapdragon_flight.md)
```

Comment:
Any reason why you put Snapdragon under complete vehicles?

Reply:
Incompetence?

All fixed now. Can you please re-review.